docs/question/html.md

```diff
@@ -1,13 +1,37 @@
-# Using Markdown in questions
+# `question.html`
 
-HTML and custom elements are great for flexibility and expressiveness. However, they're not great for working with large amounts of text, formatting text, and so on. [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a lightweight plaintext markup syntax that's ideal for authoring simple but rich text. To enable this, PrairieLearn adds a special `<markdown>` tag to questions. When a `<markdown>` block is encountered, its contents are converted to HTML. Here's an example `question.html` that utilizes this element:
+## Template file
+
+All `question.html` files are rendered with the [Mustache](https://mustache.github.io/mustache.5.html) template engine. Outside direct variable substitution, you can use Mustache to do things like conditionally render elements and render over arrays. For example, you can use the following syntax to conditionally render a piece of HTML:
```

Comment:
Is it worth including these examples at all? This is essentially reproducing limited parts of the mustache documentation, so I'm not sure how much of a value-add there is over just pointing people to the relevant sections over there.

Reply:
In my opinion it is at least worth linking to these two features on the spec, but it doesn't look like you can link to sections of that page.

Reply:
I think including a few basic examples here is a good idea.

Here's a slight rework of this (feel free to edit):

```suggestion
All `question.html` files are processed with the [Mustache template engine](https://mustache.github.io/mustache.5.html), which allows for basic dynamic rendering of HTML. For example, you can use the following syntax to conditionally render a piece of HTML:
```